lib/filewatch/since_db.rb

```diff
@@ -0,0 +1,247 @@
+require 'filewatch/boot_setup' unless defined?(FileWatch)
+
+module FileWatch
+  class SinceDb
+    def self.parse_line(line)
+      parts = line.split(" ")
+      parse_line_v2(parts) || parse_line_v1(parts)
+    end
+
+    def self.parse_line_v2(parts)
```

Comment:
can we have examples here on the kind of parsing we're doing here?
also, as before I'd rather have variable names in full, I'm too lazy to guess :)